probe/tag/pidtree.go

```diff
@@ -24,7 +24,7 @@ var (
 	readFile = ioutil.ReadFile
 )
 
-func newPIDTree(procRoot string) (*pidTree, error) {
+func newRealPIDTree(procRoot string) (*pidTree, error) {
```

Comment:
I'm not a big fan of having real in all the names...  I'm guess this is to avoid a collision with the stub for testing?  Why not call that netpidtreestub?